### ISSUE TITLE
"securityOverwrites" does not work

### ISSUE THREAD

**Issue by qtekk** [apideck-libraries/portman#472]
Hi, 
I'm trying to set Bearer Token authorization for a generated collection with "securityOverwrites", but I always get "No Auth". Am I missing something?

portman-config.json
```json
{
  "version": 1.0,
  "globals": {
    "stripResponseExamples": true,
    "securityOverwrites": {
      "bearer": {
        "token": "{{autoauth_access_token}}"
      }
    }
  }
}
```

![image](https://user-images.githubusercontent.com/17722269/224998789-4c1e3212-1018-4bef-91c4-fbc71d468eb4.png)

**Comment by thim81** [comment]
hi @qtekk 

Could it be that there is no "security" defined?
Our investigation showed that the securityOverwrites was limited to overwriting existing auth settings.

We have a [PR](https://github.com/apideck-libraries/portman/pull/474) in the making that injects auth when no auth settings are present.

**Comment by savage-alex** [comment]
@qtekk  could you share the openAPI used with the config please?

**Comment by thim81** [comment]
hi @qtekk 

We just released Portman v1.22.0, which includes improvements for the behavior of `securityOverwrites`.
Going to close this issue, if you run into the same or new problems, please let us know.

**Comment by qtekk** [comment]
Sorry I didn't respond earlier. With 'overwriteRequestSecurity' I have made a workaround by setting an authorization token for every single request.

I tested v1.22.0. And this release fixed my issue ;) The problem was indeed in missing security settings
